assets/prototype/domain/shared/predicates/datetimes/filters/belowCapacity/index.test.ts

```diff
@@ -0,0 +1,49 @@
+/**
+ * Internal dependencies
+ */
+import belowCapacity from './index';
+
+describe('belowCapacity', () => {
+	it('Should return empty array if dates are deleted', () => {
+		const filteredDates = belowCapacity({
+			capacity: 50,
+			dates: [
+				{ id: 'abc', isDeleted: true },
+				{ id: 'def', isDeleted: true },
+			],
+		});
+
+		expect(Array.isArray(filteredDates)).toBe(true);
```

Comment:
Same here related to TS.

Reply:
Cleaned up: https://github.com/eventespresso/event-espresso-core/commit/af6fd58b92aff033ddb59b2fbc9b0dcdf19261fa